src/components/UI/Layout/Navigation/SideDrawer/SideDrawer.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Drawer, ListItem } from '@material-ui/core';
+
+import { BrowserRouter as Router } from 'react-router-dom';
+import SideMenus from '../SideMenus/SideMenus';
+import SideDrawer from './SideDrawer';
+import { sideDrawerMenus } from '../../../../../config/menu';
+
+import { shallow, mount } from 'enzyme';
+
+describe('side drawer testing', () => {
+  let isToggled = false;
+  const mockCallBack = jest.fn();
+  const changeToggle = () => {
+    console.log('being called');
+    isToggled = !isToggled;
+  };
+  const component = (
+    <Router>
+      <SideDrawer MenuToggle={changeToggle} isMobile={isToggled} />
+    </Router>
+  );
+
+  it('initialized properly', () => {
+    const wrapper = mount(component);
+    expect(wrapper).toBeTruthy();
+    // Use Hidden at index 0 because there's two Drawer components.
+    expect(wrapper.find(Drawer).at(0).find(ListItem).length).toEqual(sideDrawerMenus.length);
+  });
+
+  it('callback is working', () => {
```

Comment:
This test isn't working yet. For some reason, `onClose` won't call and changed `isToggled` or print `being called`. Would love to get other insight about this as well.

Reply:
We need to restructure SideDrawer according to new requirements. This won't be needed.